Tasks/Viachaslau/Task4/query08.sql

```diff
@@ -0,0 +1,10 @@
+/*CREATE VIEW vCustOrders AS (
```

Comment:
for 1 time creation VIEW could be droped and re-created
it would be nice to specify schema name